src/pks/mpc/weak_mpc_semi_coupled_helper.cc

```diff
@@ -31,17 +31,17 @@ UpdateIntermediateStateParameters(Teuchos::RCP<Amanzi::State>& S_next_, Teuchos:
 				   S_inter_->GetField(Keys::getKey(name.str(),"conducted_energy_source"))->owner()) = 
 	     *S_next_->GetFieldData(Keys::getKey(name.str(),"conducted_energy_source"));
 	  
-	   *S_inter_->GetFieldData(Keys::getKey(name.str(),"mass_source"), 
-				   S_inter_->GetField(Keys::getKey(name.str(),"mass_source"))->owner()) = 
-	     *S_next_->GetFieldData(Keys::getKey(name.str(),"mass_source"));
```

Comment:
These are probably now DomainSets, and this needs to get updated sometime (not before release) @ajkhattak   Not sure if these MPCs are dead code or still used.